test/parallel/test-tls-dhe.js

```diff
@@ -43,7 +44,7 @@ const dheCipher = 'DHE-RSA-AES128-SHA256';
 const ecdheCipher = 'ECDHE-RSA-AES128-SHA256';
 const ciphers = `${dheCipher}:${ecdheCipher}`;
 
-if (!common.hasOpenSSL(3, 2)) {
+if (secLevel < 2) {
```

Comment:
I know this was a problem with the original code here also but it would be helpful to add a comment explaining what the `2` here is.... a pointer to the docs would suffice.